### DISCUSSION TITLE
What does this mean: "Please update your two-factor authentication methods on this page, otherwise access will be limited "?

### DISCUSSION THREAD

**Discussion by despdx** [community/community#124940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I login with my Yubikey, which I love and seems to work fine, I get prompted to do some additional 2FA authentication, which doesn't seem to work.  My TOTP auth app seems fine and it gives me a TOTP, but that code doesn't work.  When I get to my GitHub account settings page, it shows "Configured" in the 2FA section.  So, I don't know what it wants me to do.

I tried to go through the process again, but my auth app says there is already a configured account with that identity.  So confusing.  It's not clear why I need to do this, which is related to why I not sure *what* to do.

Should I just delete this identity in my auth app?  I think that will succeed, but I still don't even know if that's what GitHub wants me to do?  Did something change?  Do I need to re-do 2FA configuration regularly or something?

And why isn't my hardware key sufficient? It's way easier and more secure.  Docs say 2FA is mandatory; fine, but hardware key *is* 2FA. That's part of my confusion.

**Comment by shivamgaur99** [comment]


**Reply by despdx** [reply]
Thanks, but I didn't find that helpful.

BTW, I tried to contact support, that's how I ended up here.  It's very difficult.

**Comment by despdx** [comment]
Aha!  I have to YubiKeys and different GitHub accounts configured with different keys.  Confusing, but I can live with it. Okay, mystery solved, at least.
